samples/wire/README.md

```diff
@@ -405,12 +405,12 @@ There is support for [binding interfaces][interfaces], [binding
 values][values], as well as support for [cleanup functions][cleanup]. See the
```

Comment:
Should this whole sample be moved to the `wire` repo? It seems to be all about wire.

Fine to do in a separate PR.

Reply:
Agree to separate the sample, but we'll want some pointers in this repo to Wire as well.

Reply:
Agreed. Opened #836 to track removing the sample from this repository.